docs-gen/includes/quickstart/pyproject.toml

```diff
@@ -0,0 +1,12 @@
+[tool.poetry]
+name = "quickstart"
+version = "0.1.0"
+description = ""
+authors = ["Hephaestus Team <DL-Discovery-Hephaestus@sky.uk>"]
```

Comment:
We shouldn't expose this sky internal team, please

Reply:
done